packages/dotnet-sdk/README.md

```diff
@@ -38,8 +38,9 @@ For .NET 6 projects (VS 2022): Choose version >= 0.3.0-rc.
 "TeamsFx": {
     "Authentication": {
         "ClientId": "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx",
-        "SimpleAuthEndpoint": "https://localhost:44357/",
-        "InitiateLoginEndpoint": "https://localhost:44357/auth-start.html"
+        "ClientSecret": "xxx", // 'User Secrets' is a better place to store secret string.
+        "InitiateLoginEndpoint": "https://localhost:44357/auth-start.html",
```

Comment:
Our future default port is 44302, avoid 44357 to cause some unexpected problem.

Reply:
Thanks for the reminder.